source/app/components/Button/Button.module.scss

```diff
@@ -68,7 +68,7 @@ $button-palettes: (
 
 // Palette specific styles
 @each $palette, $colors in $button-palettes {
-  .#{$palette} {
+  .#{"" + $palette} {
```

Comment:
Just curious, what impact does this change bring in?

Reply:
SCSS compiler detects this as a code smell essentially. I trust that it's correct so do this to silence the warning.